Selenium_Basics_2/Selenium_Basics_2/EpamSiteTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Selenium_Basics_2;
+
+public class EpamSiteTest
+{
+    private IWebDriver driver;
+    private const string urlMain = "https://www.epam.com/";
+
+    [SetUp]
+    public void Setup()
+    {
+        driver = new ChromeDriver();
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(urlMain);
+        driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(20);
+        Thread.Sleep(10000);
+        driver.FindElement(By.XPath("//button[@id='onetrust-accept-btn-handler']")).Click();
+    }
+
+    [Test]
+    public void CheckListAvailableCountries()
+    {
+        List<string> webElemList = new List<string>() { "AMERICAS", "EMEA", "APAC" };
+        driver.FindElement(By.XPath("//*[@href='/careers']")).Click();
+        var actualCountriesList = driver.FindElements(By.XPath("//*[@class='tabs__title' and contains(@role,'presentation')]")).Select(item => item.Text);
+
+        Assert.That(actualCountriesList, Is.EqualTo(webElemList), "List of countries doesn't have: Americas, EMEA, APAC");
+    }
+
+    [Test]
+    public void CheckSearchInputAutomation()
+    {
+        string urlSearch = "https://www.epam.com/search?q=Automation";
+
+        driver.FindElement(By.XPath("//*[@class='header-search-ui header__control']")).Click();
+        driver.FindElement(By.Id("new_form_search")).SendKeys("Automation");
+        driver.FindElement(By.XPath("//form[@action='/search']/child::button[@class='header-search__submit']")).Click();
+
+        Assert.That(driver.Url, Is.EqualTo(urlSearch), "Search doesn't work!");
+        bool isContains = driver.PageSource.Contains("Automation");
```

Comment:
you should have take 5 first articles and check their text

Reply:
Fixed